pubspec.yaml

```diff
@@ -12,9 +12,9 @@ dependencies:
   flutter:
     sdk: flutter
   # package_info will help us to find current installed version of application.
-  package_info: ^2.0.0
+  package_info: ^2.0.2
```

Comment:
changes in dependencies need to be justified and deserve and individual commit. Only if the dependencies update are needed to allow the changes, then they should be together but this does not seem to be the case.